doc/examples/plot_seam_carving.py

```diff
@@ -0,0 +1,94 @@
+"""
+============
+Seam Carving
+============
+
+This example demonstrates how images can be resized using seam carving [1]_.
+Resizing often distorts contents in the image. Seam carving tries to resize
```

Comment:
@JDWarner It doesn't remove visually similar regions.  It removes regions with less importance  (according to the cost matrix given ).